### ISSUE TITLE
No type hints displayed (turned out to be large latency)

### ISSUE THREAD

**Issue by mrl5** [fannheyward/coc-rust-analyzer#1068]
**What's the output of `:CocInfo`**
```
## versions

vim version: VIM - Vi IMproved 8.2 8022434
node version: v18.11.0
coc.nvim version: 0.0.82-f39f5f69 2022-10-07 23:24:25 +0800
coc.nvim directory: /home/kuba/.vim/plugged/coc.nvim
term: dumb
platform: linux

## Log of coc.nvim

2022-10-15T13:59:29.810 INFO (pid:156730) [services] - registered service "rust-analyzer"
2022-10-15T13:59:29.811 INFO (pid:156730) [services] - LanguageClient Rust Analyzer Language Server state change: stopped => starting
2022-10-15T13:59:29.811 INFO (pid:156730) [plugin] - coc.nvim initialized with node: v18.11.0 after 111ms
2022-10-15T13:59:29.822 INFO (pid:156730) [language-client-index] - Language server "rust-analyzer" started with 156753
2022-10-15T13:59:29.832 INFO (pid:156730) [services] - LanguageClient Rust Analyzer Language Server state change: starting => running
2022-10-15T13:59:29.838 INFO (pid:156730) [services] - service rust-analyzer started
2022-10-15T13:59:31.315 INFO (pid:156730) [attach] - receive notification: showInfo []
```

**What's the output of `:CocCommand rust-analyzer.serverVersion`**
```
[coc.nvim] rust-analyzer 1.64.0 (a55dd71 2022-09-19)
```

**What's your coc-rust-analyzer version? You can get it from `:CocList extensions`**
```
coc-rust-analyzer 0.69.1
```

## additional context
just `coc-rust-analyzer`
![coc-rust-analyzer](https://i.ibb.co/k2Sq00b/Screenshot-2022-10-15-14-04-21.png)

after I `:CocInstall coc-rls` - `coc-rust-analyzer` + `coc-rls`
![coc-rust-analyzer and coc-rls](https://i.ibb.co/2Pxzf2X/Screenshot-2022-10-15-14-07-25.png)

and then `:CocInfo` is
```
## versions

vim version: VIM - Vi IMproved 8.2 8022434
node version: v18.11.0
coc.nvim version: 0.0.82-f39f5f69 2022-10-07 23:24:25 +0800
coc.nvim directory: /home/kuba/.vim/plugged/coc.nvim
term: dumb
platform: linux

## Log of coc.nvim

2022-10-15T14:06:51.008 INFO (pid:163493) [services] - registered service "rust-analyzer"
2022-10-15T14:06:51.009 INFO (pid:163493) [services] - LanguageClient Rust Analyzer Language Server state change: stopped => starting
2022-10-15T14:06:51.020 WARN (pid:163493) [workspace] - workspace.createStatusBarItem is deprecated, please use window.createStatusBarItem instead. 
    at Object.<anonymous> (/home/kuba/.config/coc/extensions/node_modules/coc-rls/lib/index.js:5711:39)
    at __webpack_require__ (/home/kuba/.config/coc/extensions/node_modules/coc-rls/lib/index.js:20:30)
2022-10-15T14:06:51.020 WARN (pid:163493) [workspace] - workspace.createOutputChannel is deprecated, please use window.createOutputChannel instead. 
    at Object.<anonymous> (/home/kuba/.config/coc/extensions/node_modules/coc-rls/lib/index.js:131:44)
    at Generator.next (<anonymous>)
2022-10-15T14:06:51.022 INFO (pid:163493) [services] - registered service "rust-client"
2022-10-15T14:06:51.023 INFO (pid:163493) [plugin] - coc.nvim initialized with node: v18.11.0 after 124ms
2022-10-15T14:06:51.079 INFO (pid:163493) [extension:coc-rls] - Detected active channel: stable-x86_64-unknown-linux-gnu (since 'rust-client.channel' is unspecified)
2022-10-15T14:06:51.090 INFO (pid:163493) [language-client-index] - Language server "rust-analyzer" started with 163520
2022-10-15T14:06:51.106 INFO (pid:163493) [services] - LanguageClient Rust Analyzer Language Server state change: starting => running
2022-10-15T14:06:51.111 INFO (pid:163493) [services] - service rust-analyzer started
2022-10-15T14:06:51.163 INFO (pid:163493) [extension:coc-rls] - Detected active channel: stable-x86_64-unknown-linux-gnu (since 'rust-client.channel' is unspecified)
2022-10-15T14:06:51.193 INFO (pid:163493) [services] - LanguageClient Rust Language Server state change: starting => running
2022-10-15T14:07:04.591 WARN (pid:163493) [completion-complete] - Complete timeout after 5000ms [ 'rust-analyzer' ]
2022-10-15T14:09:29.939 INFO (pid:163493) [attach] - receive notification: showInfo []
```

I've seen #1057 but I guess it's not a case for me

**Comment by mrl5** [comment]
what am I missing here? maybe:
```
2022-10-15T14:06:51.022 INFO (pid:163493) [services] - registered service "rust-client"
```

**Comment by fannheyward** [comment]
You should use coc-rls any more.

**Comment by mrl5** [comment]
@fannheyward that's my point! without `coc-rls` it doesnt work and I want to use `coc-rust-analyzer` only

**Comment by fannheyward** [comment]
Can't reproduce. `:CocUninstall coc-rls` and try again.

**Comment by mrl5** [comment]
ok, I did and after I uninstalling `coc-rls` it's like in first screenshot

**Comment by fannheyward** [comment]
Set `"rust-analyzer.trace.server": "verbose"` in your coc-settings.json, open an rust project, then `:CocCommand workspace.showOutput rust-analyzer` will show trace logs, any errors?

**Comment by mrl5** [comment]
I got it working and the reason is awful latency. To be 100% sure I also tested inside `cargo init foobar` and indeed `coc-rust-analyzer` works on my current setup.

When it comes to the codebase where I was checking: it eventually started giving hints for `Uuid`. I wonder why `coc-rls` is much more faster than `coc-rust-analyzer`?

BTW is there a way to enable this "indexing" popups/indicators that were present in previous versions? (idk if this is `coc-rust-analazyer` thing or just `coc.nvim`
